projects/plugins/jetpack/_inc/client/components/jetpack-manage-banner/index.jsx

```diff
@@ -0,0 +1,83 @@
+import { getRedirectUrl, UpsellBanner } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import analytics from 'lib/analytics';
+import PropTypes from 'prop-types';
+import React, { useCallback } from 'react';
+import jetpackManageIcon from './jetpack-manage.svg';
+
+const JetpackManageBanner = props => {
+	const trackEvent = useCallback(
+		target => {
+			analytics.tracks.recordJetpackClick( {
+				target: target,
+				feature: 'manage',
+				page: props.path,
+			} );
+		},
+		[ props.path ]
+	);
+
+	const handleLearnMoreClick = useCallback( () => {
+		trackEvent( 'jp-manage-learn-more-click' );
+	}, [ trackEvent ] );
+
+	const handleManageSitesClick = useCallback( () => {
+		trackEvent( 'jp-manage-dashboard-sites-click' );
+	}, [ trackEvent ] );
+
+	const handleSignUpForFreeClick = useCallback( () => {
+		trackEvent( 'jp-manage-sign-up-click' );
+	}, [ trackEvent ] );
+
+	const redirectOrigin = 'jetpack-at-a-glance';
```

Comment:
For something like this, I'd prefer not programmatically building the URL so that it's easier to search.

Reply:
Yep, I agree, and now I recall Renzo's comment about it.